lib/vbox/dialog.go

```diff
@@ -112,3 +123,67 @@ func (v *Config) USBDialog() {
 		v.Option.USB.USBType.XHCI = onoff()
 	}
 }
+
+// Creates custom virtualbox specs
```

Comment:
__[golint]__
* [golint] comment on exported function SetVbox should be of the form "SetVbox ..."

<!-- Comment by [SideCI](https://sideci.com) -->